.github/CONTRIBUTION.md

```diff
@@ -19,11 +19,25 @@ Run lint with:
 
 `yarn lint`
 
-## Publishing
+## Releasing
 
-Currently only admins are able to publish.
+Currently only admins are able to create a release. A release consists of the following:
 
-1. Check out master and make sure it is up to date
-2. Run `npm version minor/patch`
-3. Verify changes in `package.json`, commit and run `git push && git push --tags`
-4. Make sure all checks pass, then Circle automatically publishes to NPM.
+- Bumping the package version based on your commits
+- Updating the API specification
+- Creating a new commit with the changed files
+- Creating a tag with the new version
+- Pushing the release commit and tag to master
+
+### Step-By-Step
+
+1. Check out master and run `git pull`.
+1. Run `git clean -dfx && yarn` to make sure depenencies are up-to-date.
+1. Run `npm version [major | minor | patch] -m "chore(release): v%s"`. Use semver string based on conventional commits since last release. Ex: `npm version patch -m "chore(release): v%s"`.
+1. Run `git push && git push --tags` to push commit and tag.
+1. Make sure all checks pass, then Circle CI automatically publishes to NPM.
+
+### On version command failure
```

Comment:
I do not understand this section

Reply:
Actually good catch. This section doesn't make sense in the context of failed npm version command. You should not end up in a scenario where a tag is created if `preversion` or `version` scripts fails.

But it should perhaps be updated to include instructions on how to revert `npm version` if one is for some reason not happy with the end result.

Reply:
For example, if a version number already exists on your machine, npm version will fail. Then you have to remove it and try again

Reply:
I see. Thanks. Could you also add those explanations into the docs?

Reply:
OK I can clarify a bit.

Reply:
fixed